semeio/workflows/localisation/local_script_lib.py

```diff
@@ -294,6 +296,42 @@ def apply_decay(
     return scaling_values
 
 
+def apply_constant(
+    row_scaling,
+    data_size,
+    grid,
+    value,
+    log_level,
+    calculate_qc_parameter=False,
+):
+    # pylint: disable=too-many-arguments,too-many-locals
+    """Assign constant value to the scaling factor,
+    assign it to ERT instance by row_scaling
+    and returns a full sized grid parameter with scaling factors for active
+    grid cells and 0 elsewhere to be used for QC purpose.
+    """
+    debug_print(f"Scaling factor is constant: {value}", LogLevel.LEVEL3, log_level)
+    decay_obj = ConstantScalingFactor(value)
+
+    scaling_vector = np.zeros(data_size, dtype=np.float32)
```

Comment:
This is unfamiliar territory for me, so I'll just start poking around and hopefully eventually understand whats going on.

First question, why do we use np.float32 here?